### PR TITLE
add super-convergence to apphub

### PR DESCRIPTION
**Pull request by geez0219**

### CODE REVIEW

**Comment by vbvg2008** [comment]
need the real acc vs step graph.

File: apphub/lr_controller/super_convergence/super_convergence_tf.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
```
Comment (TortoiseHam) [inline apphub/lr_controller/super_convergence/super_convergence_tf.py:1]: 2021

File: apphub/lr_controller/super_convergence/super_convergence_tf.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    x = layers.BatchNormalization(momentum=0.8)(x)
+    x = layers.LeakyReLU(alpha=0.1)(x)
+    x = layers.Conv2D(num_channel, 3, padding='same')(x)
+    x = layers.BatchNormalization(momentum=0.8)(x)
+    x = layers.LeakyReLU(alpha=0.1)(x)
+    return x
+
+
+def my_model():
+    # prep layers
```
Comment (TortoiseHam) [inline apphub/lr_controller/super_convergence/super_convergence_tf.py:64]: if we're going to keep re-using this model, should we add it to the FE architecture directory?

File: apphub/lr_controller/super_convergence/super_convergence_tf.py (hunk 1 above)
Comment (vbvg2008) [reply apphub/lr_controller/super_convergence/super_convergence_tf.py:1]: nice catch!

File: apphub/lr_controller/super_convergence/super_convergence_tf.py (hunk 2 above)
Comment (geez0219) [reply apphub/lr_controller/super_convergence/super_convergence_tf.py:63]: Maybe? Is this a well-known model?

Comment (vbvg2008) [reply apphub/lr_controller/super_convergence/super_convergence_tf.py:63]: I created a task in project board.

**Review by vbvg2008 — APPROVED** [review]

**Pull request merged**